docs/visualization/visualize_unit_metrics.ipynb

```diff
@@ -107,14 +107,14 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # dandi_api_key = os.environ['DANDI_API_KEY']
</pre><p>Not sure how the notebook ran with GitHub actions with this commented</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>This is because <code>authenticate</code> is set to false, and the dandiset is visual coding (unembargoed) </p>